### DISCUSSION TITLE
can fairy stockfish be configured to play capture anything chess?

### DISCUSSION THREAD

**Discussion by sim-museum** [fairy-stockfish/Fairy-Stockfish#775] [Q&A]
The [capture anything](https://www.chess.com/terms/capture-anything-chess) chess variant where you can capture your own pieces was praised by [Vladimir Kramnik](https://www.chess.com/players/vladimir-kramnik)  in the study of chess variants he wrote with [DeepMind](https://www.chess.com/news/view/new-alphazero-paper-explores-chess-variants). 

Can fairy stockfish be configured to play capture anything chess?

**Comment by ianfab** [comment]
Not yet. This feature request is already tracked at #608 though.
